module-3/pop/HomePage.js

```diff
@@ -7,16 +7,31 @@ const Layout = require('./Layout');
 const Element = require('./Element');
 const Elements = require('./Elements');
 
-module.exports = class HomePage extends Layout {
-    constructor() {
-        super('/');
-
-        this.header = new Element('Header', by.css('header'));
-        this.header.addChildren(new Element("Logo", by.css('.header__logo')));
-        this.menuItems = new Elements("Menu Items", by.css('.top-navigation__item-link'));
+class HomePage extends Layout {
+    constructor(name, url, locator) {
+        super(name, url, locator);
+        
+        this.newElementHeader = new Element('Header', {css:'header'});
+        this.newElementLogo = new Element('Logo', {css:'.header__logo'});
+        this.newElementMenuItems = new Elements("Menu Items", {css:'.top-navigation__item-link'});
+        this.newElementFooter = new Elements("Footer", {css:'.footer__holder'});
     }
 
-    get logo() {
-        return this.get('Logo');
+    getLogo () {
```

Comment:
Getters named like this should return the object itself not just a property, either rename the function, or change the return value